keps/sig-api-machinery/3903-unknown-version-interoperability-proxy/README.md

```diff
@@ -258,7 +278,12 @@ TODO: security / cert stuff.
 
 ## Design Details
 
-TODO: specific API change (x2)
+We need two API changes:
+
+1. The apiservices API needs to be augmented with a `serviceableBy` field, which
```

Comment:
I'm unsure we need this now! StorageVersion gives us apiserver id choices for a GVR, no?